aws-java-sdk-core/src/main/java/com/amazonaws/retry/PredefinedRetryPolicies.java

```diff
@@ -199,6 +198,10 @@ public boolean shouldRetry(AmazonWebServiceRequest originalRequest,
         /** Maximum exponential back-off time before retrying a request */
         private static final int MAX_BACKOFF_IN_MILLISECONDS = 20 * 1000;
 
+        /** Maximum number of retries before the max backoff will be hit */
+        private static final int MAX_RETRIES_BEFORE_MAX_BACKOFF =
```

Comment:
ok, if I do that I'll also add a comment on the other constants about how people need to update this new one if they update the others and give them the formula I used for it.  Is that better?  I'm definitely not committing the magic number 6 with no context.